### ISSUE TITLE
SPMC 17 Alpha 11 - 4K

### ISSUE THREAD

**Issue by CLFNole** [koying/SPMC#1065]
I had switched to Alpha 11 and was watching some 4K movies and noticed a lot of buffering. After a while I switched back to Alpha 10 and had no buffering or just brief buffering with several 4K addons after. Was anything changed from 10 to 11 or did anyone else notice this? Perhaps it was just coincidental.

**Comment by ghost** [comment]
I am noticing buffering problems using a11 as well, but I also just changed my network setup.

I'm using an Nvidia Shield TV. Is there any way to measure my connection to my NAS?

**Comment by ziggy73701** [comment]
Also seeing buffering in HEVC encoded 4K which wasn’t there in Alpha 10.
This is a local file, no addons involved.

**Comment by CLFNole** [comment]
Anyone still experiencing 4K buffering with They new 17.6 betas?  I tried it and still had buffering then went back to 17.4 Alpha 10 and basically no 4K buffering.

**Comment by satwise** [comment]
Yes, also experiencing more buffering since installing 17.4a11. my Nvidia Shield also has the latest updates and gigabit hardwired movie files over mine as are very slow as well especially 4K HDR which incidentally is no longer detected by my LG OLED since 17.6a
